### PR TITLE
Add Forgot password Page

### PR DESCRIPTION
**Pull request by SubhamB2003**
# Description
solves #238 

@thestarsahil Please check my prs

Please include a summary of the change and which issue is fixed. List any dependencies that are required for this change.

## Fixes #(issue_no)

Replace `issue_no` with the issue number which is fixed in this PR

## Checklist

- [ ] Tests have been added or updated to cover the changes
- [ ] Documentation has been updated to reflect the changes
- [ ] Code follows the established coding style guidelines
- [ ] All tests are passing

### CODE REVIEW

**Comment by thestarsahil** [comment]
Screenshot 2023-07-16 105835

Conflict occurs

**Comment by SubhamB2003** [comment]
@thestarsahil Please check.

**Comment by thestarsahil** [comment]
Check the codebase again i marked it where to change

**Comment by SubhamB2003** [comment]
> Check the codebase again i marked it where to change

Hey, I removed below two-element
image

**Review by thestarsahil — CHANGES_REQUESTED** [review]

**Comment by thestarsahil** [comment]
You haven't checked yet properly , Have a looked on changes

File: src/components/Login/Login.jsx [hunk 1]
```diff
@@ -1,18 +1,16 @@
...
-import hide from "../../assets/hide.png";
-import meeting from "../../assets/meeting.png";
-import microsoft from "../../assets/microsoft.png";
-import googlePlay from "../../assets/google-play.png";
-import { auth,googleAuthProvider } from "../../firebase/auth";
 import {
   signInWithEmailAndPassword,
   signInWithPopup
 } from "firebase/auth";
+import React, { useEffect, useState } from "react";
```
[the remark this answers is not in the document]
Comment (SubhamB2003) [reply src/components/Login/Login.jsx:5]: Hey, I am not changing anything I just format the document that's why it showing changes

File: src/components/Login/Login.jsx [hunk 2]
```diff
@@ -58,14 +56,14 @@ export default function Login() {
...
       signInWithEmailAndPassword(auth, email, password)
-      .then(() => {
-            navigate("/dashboard");
-      })
-      .catch((err) => {
-        if(err == "FirebaseError: Firebase: Error (auth/wrong-password)."){
-          seterror("Incorrect Password!");
-        }
-      });
+        .then(() => {
```
[the remark this answers is not in the document]
Comment (SubhamB2003) [reply src/components/Login/Login.jsx:59]: same as this also

**Comment by SubhamB2003** [comment]
@thestarsahil Please review my pr

**Comment by thestarsahil** [comment]
Hi Subham,

I just finished checking the codebase structure, and it looks great! Thank you so much for your wonderful contribution. Your work has been very helpful, and I appreciate your attention to detail.

I'm particularly impressed with the way you organized the code.
Overall, I'm very happy with your work.

**Comment by SubhamB2003** [comment]
> Hi Subham,
> 
> I just finished checking the codebase structure, and it looks great! Thank you so much for your wonderful contribution. Your work has been very helpful, and I appreciate your attention to detail.
> 
> I'm particularly impressed with the way you organized the code.
> Overall, I'm very happy with your work.

Thank you😊,
Happy to contribute

**Pull request merged**